assets/illustrations/directory.svg

```diff
@@ -0,0 +1,41 @@
+<svg width="148" height="219" viewBox="0 0 148 219" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Might be a good idea for us to use https://jakearchibald.github.io/svgomg/ when Tsui corrects the SVG grouping issue for us, it minimises the storage size of the SVGs

Reply:
Ohh this is really cool! We can deffo use